scripts/logs.sh

```diff
@@ -35,8 +35,11 @@ rsync -azr /var/log/staging/logs.tar.gz node@hub.goddard.unicore.io:/var/log/nod
 # delete all the old logs now
 rm /var/log/nginx/*.log
 
+# Manually tell Nginx to rotate the logs to ensure new ones are created after removal.
+kill -USR1 `cat /var/run/nginx.pid`
```

Comment:
This compared to a nginx reload ? Just wondering what the process if for NGINX here ?